packages/features/ee/workflows/lib/reminders/templates/emailRatingTemplate.ts

```diff
@@ -79,3 +79,6 @@ const emailRatingTemplate = ({
 };
 
 export default emailRatingTemplate;
+
+export const plainTextTemplate =
```

Comment:
For tests, it wouldn't make sense to strip the templates twice with the same function since it would always pass. By adding the plain text templates, we can detect whether the template was changed or something changed with the comparison function.